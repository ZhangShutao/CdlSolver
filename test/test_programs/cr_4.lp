% representing the preference relation between cr-rules, failed

:- a, b.
:- not a, not b.
a :- apply(r1).
b :- apply(r2).

apply(r1) :- not &c{not apply(r1)}.
apply(r2) :- not &c{not apply(r2)}, not apply(r1).