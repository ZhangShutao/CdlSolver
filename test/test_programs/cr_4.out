default model: 1
x:b, apply(r2); y:b, apply(r2);
default model: 2
x:apply(r1), a; y:apply(r1), a;
Satisfiable